doc/source/computation.rst

```diff
@@ -428,15 +428,13 @@ Using a non-regular, but still monotonic index, rolling with an integer window d
 
 .. ipython:: python
 
-
-   dft = DataFrame({'B': [0, 1, 2, np.nan, 4]},
-                   index = pd.Index([pd.Timestamp('20130101 09:00:00'),
-                                     pd.Timestamp('20130101 09:00:02'),
-                                     pd.Timestamp('20130101 09:00:03'),
-                                     pd.Timestamp('20130101 09:00:05'),
-                                     pd.Timestamp('20130101 09:00:06')],
-                                    name='foo'))
-
+   dft = pd.DataFrame({'B': [0, 1, 2, np.nan, 4]},
+                      index = pd.Index([pd.Timestamp('20130101 09:00:00'),
```

Comment:
It *was* broken :-) (as I noticed in the travis doc build log, otherwise I wouldn't have catched this)